src/briefcase/__main__.py

```diff
@@ -41,8 +42,8 @@ def main():
         with suppress(KeyboardInterrupt):
             logger.save_log_to_file(command)
 
-    sys.exit(result)
+    return result
```

Comment:
This makes it easier to test the result, as you can test the return value, rather than catching the `sys.exit()`